meetings/2021-09-01.md

```diff
@@ -0,0 +1,107 @@
+
+The top level of Read and Write are for sure very solid. 
+Indeed, I have [implemented a Web Server](https://github.com/bblfish/lens-play/blob/master/src/main/scala/server/Server.scala) using 
+a very simple data structure for containers, together with a concept from Functional Programming 
+called Lenses (see [An Introduction to Lenses in JS](https://medium.com/javascript-inside/an-introduction-into-lenses-in-javascript-e494948d1ea5)). 
+Lenses are essentially built out of two types of operations: getter and setters;
+or Read and Write operations. With those operations, one can build up all
+an LDP web server. 
+Having said that about Read/Write, I think Control is in the wrong place. 
+I don't see it as type of mode, I see it as relation.
+
+**Eric** I think this is just a linguistic confusion. Control just stands for edit the 
+access control resource.
+
+**Sarven** Some other ACL mechanisms model control, they also use term owner. Google's ACL uses owner.
+
+**Pavlik** 
+
+**Sarven** In WAC we treat ACL as data points as well. Even possibility of ACL having ACL suggests that they should be considered as data point. 
+I agree that it can be achieved in a different way. The aspect of operations has been in the protocol for quite a while. 
+Operations are not just atomic modes but class of operations e.g. write includes create and delete. See https://github.com/solid/web-access-control-spec/issues/85 .
+I talked to Kjetilk about why create the operation issue under authorization-panel - it was to create engagement. Protocol has some of those definitions 
+of operations and WAC currently defines mapping of HTTP methods to access modes only to inform developers. 
+Currently, operations are normative whereas specific HTTP methods are non-normative. 
+The reason is eg. PUT typically needs Write mode, but PUT + If-None-Match: * would need Write and Read.
+
+**Henry** 
+  To Eric's point above: the reason I am reaching out to Category Theory, is that 
+it is the mathematics of structures. What it tells us is that Dynamic Systems can be
+modelled using essentially just these Read and Write operations. See the maths
+discussed on [web-cats](https://gitlab.com/web-cats/CG/-/issues/28#note_666179055). 
+Since we can also do [without Control mode](https://github.com/solid/web-access-control-spec/issues/94), 
+that indicates that it is not a structural necessary piece at the level of Read and Write.
+It should be modelled differently (via the `Link` header).
+  
+**Sarven** PATCH can include DELETE, so in that case it will require read and write.
+
+**Pavlik** can we have Write without Read?
+
+**Pavlik** How do we see that List mode, can we separate reading container description and the list of containment triples?
```

Comment:
```suggestion
**Sarven** `PATCH` can include `DELETE`, so in that case it will require `read` and `write`.

**Pavlik** can we have `Write` without `Read`?

**Pavlik** How do we see that `List` mode? can we separate reading container description and the list of containment triples?
```